pkgs/applications/networking/browsers/mullvad-browser/default.nix

```diff
@@ -228,7 +228,7 @@ stdenv.mkDerivation rec {
     updateScript = callPackage ../tor-browser-bundle-bin/update.nix {
       inherit pname version meta;
       baseUrl = "https://cdn.mullvad.net/browser/";
-      prefix = "mullvad-browser-";
+      name = "mullvad-browser";
```

Comment:
This doesn't belong to this tor-browser PR.

Reply:
Well, I changed the argument in the update script, so this change is needed for it not to break.

Reply:
Ah, now I see. Sorry for the noise.